packages/discovery/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@l2beat/discovery",
   "description": "L2Beat discovery - engine & tooling utilized for keeping an eye on L2s",
-  "version": "0.4.0",
+  "version": "0.5.0",
```

Please do not bump the version manually
```suggestion
  "version": "0.4.0",
```

---

Forgot about semi-automed (:trollface:) deployments